standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/aggr/LongColumnStatsAggregator.java

```diff
@@ -192,7 +207,13 @@ public ColumnStatisticsObj aggregate(List<ColStatsObjWithSourceInfo> colStatsWit
               csd.setLongStats(aggregateData);
               adjustedStatsMap.put(pseudoPartName.toString(), csd);
               if (useDensityFunctionForNDVEstimation) {
-                densityAvgSum += ((double) (aggregateData.getHighValue() - aggregateData.getLowValue())) / aggregateData.getNumDVs();
+                if (aggregateData.getNumDVs() == 0) {
```

Comment:
could it be `0` here as well?
https://github.com/apache/hive/pull/5674/files#diff-004a5c809d016b31028b52e5a5e591d30b446d974b0d22e386c35acfee845d59L227